bucket/tflint.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/terraform-linters/tflint",
+    "description": "A Terraform linter focused on possible errors, best practices, etc.",
+    "license": "Mozilla Public License 2.0",
+    "version": "0.13.1",
+    "architecture": {
```

Comment:
```suggestion
    "depends": "terraform",
    "architecture": {
```

Reply:
Why is this depend needed in first time?

I can check simple alfa.tf with reference to variables.tf without any issue.

![image](https://user-images.githubusercontent.com/13260377/69331779-92a10100-0c55-11ea-807e-7691519ba817.png)


Only issue is when I change POLICY to POLICYa

![image](https://user-images.githubusercontent.com/13260377/69331650-70a77e80-0c55-11ea-9be4-2735bd8dd89f.png)

Reply:
@Ash258 I was working from the docs so I missed this. It should be removed by your first requested change.